lib/spack/spack/test/cmd/env.py

```diff
@@ -2537,58 +2537,88 @@ def test_stack_view_no_activate_without_default(
         assert viewdir not in shell
 
 
+@pytest.mark.parametrize("include_views", [True, False, "split"])
```

Comment:
I have a suspicion that this test could be simpler if we read/write yaml, but this does seem to be a correct test so I'm not too worried about it. Just takes quite a while to grok.